tests/test_smart_task.py

```diff
@@ -14,6 +14,7 @@
         "15013": [
             {"5712": 18000, "5851": 254, "9003": 65537},
             {"5712": 18000, "5851": 254, "9003": 65538},
+            {"5712": 19000, "5851": 230, "9003": 65539},
```

Comment:
Wasn't sure if the `devices_list` property worked properly so I added one more record here.